nconf/args_test.go

```diff
@@ -37,7 +37,7 @@ PF_LOG_QUOTE_EMPTY_FIELDS=true
 		EnvFile: tmp.Name(),
 	}
 
-	log, err := args.Setup(cfg, "")
+	log, err := args.Setup(cfg, "", "")
```

Comment:
maybe we should also add tests to check whether the global tracer and metrics have been configured successfully

Reply:
I was looking into that a bit ~ It isn't actually easy to access those in their packages. For instance, there is no way to get the `Sink` in the metriks package. I didn't think it worth the effort personally.